skycoin/dmsg-deprecated
# fixed messages

Changes:
- Printing the right error message
	
How to test this PR:
Now it should show the right err message when there is a failure communicating with discovery. Before, if code broke at those points the error message was always `err something unexpected happened`.

For example if you use `(*client).Entry(...)` now requesting an non-existent PK you should be able to see an error message telling you that such key doesn't exists.
